src/components/pages/Publications/PublicationsConstants.ts

```diff
@@ -27,6 +46,8 @@ const listOfReports: PublicationsCardProps[] = [
         img: InventoryEvidenceSynthesis,
         titleKey1: 'IndustryReportTitles',
         titleKey2: ['InventoryEvidenceSynthesis'],
+        publicationName: "",
```

Comment:
Instead of defining these as blank strings when a card has no publicationName or author, I think we should just make these optional props and just not supply them